### PR TITLE
Attestation Examples added

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -944,7 +944,7 @@ of extended claims.
 
 As of the writing of this specification, device and TEE attestations have not been standardized
 across the market. Different devices, manufacturers, and TEEs support different attestation
-algorithms and mechanisms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
+algorithms and mechanisms (for example, ECDSA-based attestation, Direct Anonymous Attestation (DAA), Enhanced Privacy ID (EPID), a Privacy CA). In order for TEEP to be inclusive, it is agnostic to the format of evidence,
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]: I think we would need informative references for each of these, to accept this PR

**Review by dthaler — CHANGES_REQUESTED** [review]

File: draft-ietf-teep-architecture.md (hunk 1 above)
Comment (hannestschofenig) [reply draft-ietf-teep-architecture.md]: I thought about this again and came to the conclusion that this document is not the right place to discuss attestation techniques.

File: draft-ietf-teep-architecture.md [hunk 2]
```diff
@@ -944,13 +944,14 @@ of extended claims.
...
-algorithms and mechanisms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
+algorithms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
 allowing proprietary or standardized formats to be used between a TEE and a verifier (which may or may not
 be colocated in the TAM). However, it should be recognized
 that not all Verifiers may be able to process all proprietary forms of attestation evidence.
 Similarly, the TEEP protocol is agnostic as to the format of attestation results, and the protocol
 (if any) used between the TAM and a verifier, as long as they convey at least the required set of claims
-in some format.
+in some format. Note that the respective attestation algorithms are not defined in TEEP protocol itself; the 
+impact is on the attestation token format used within the TEEP protocol instead.
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
in some format. Note that the respective attestation algorithms are not defined in the TEEP protocol itself; see {{I-D.ietf-rats-architecture}} and {{I-D.ietf-teep-protocol}} for more discussion. 
```

File: draft-ietf-teep-architecture.md [hunk 3]
```diff
@@ -944,13 +944,14 @@ of extended claims.
 
 As of the writing of this specification, device and TEE attestations have not been standardized
 across the market. Different devices, manufacturers, and TEEs support different attestation
-algorithms and mechanisms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
+algorithms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
protocols. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
```

**Pull request merged**